### ISSUE TITLE
Missed slam

### ISSUE THREAD

**Issue by ThorvaldAagaard** [EdwardPiwowar/BBA#355]
08232505FC9249F94481FE8ECA56
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/26640ac1-f77d-41e1-9f1d-486456e1106b)

I think the North hand is worth one more try.

Splinter with A or K is mostly bad, but it looks like North can only make a slam try by showing a splinter.

**Comment by ThePokerDude** [comment]
Do we have any convention to show 6h M and slam interest (independent of shortness).
Direct 4d maybe?
This would be better than 2d followed by splinter. South would know that a hand with a lot of controls should bid on.

**Comment by ThorvaldAagaard** [comment]
No agreements - I have been using the other major for showing a hand with slam-interest, that works fine with Smolen.
Also remember, that transfer to a major and bidding game is a mild slam-try when playing Texas as we do.

And for the more advanced players you can play transfer after the major transfer, then you have space to distinguish between Cue and splinter slam try, and balanced/unbal choice of contract, but this is to advanced for BEN currently.

![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/bb4dbf1d-d21f-4d52-98c2-f60e49daea93)

**Comment by ThorvaldAagaard** [comment]
But I think Edward forgot that when playing Texas, the transfer like here
29ADB3241308EEE6183A72B44A1D
![image](https://github.com/EdwardPiwowar/BBA/assets/10560652/1d405046-232a-4085-be88-99f320d36d6e)
is wrong. as North has no slam interest, and the 4S is wrongly explained

**Comment by EdwardPiwowar** [comment]
1. I don't know how N can find out more
```
          AKQ
          Q97643
          983
          A
5432                 JT9
A2                   JT8
42                   KJT7
JT432                987
          876
          K5
          AQ65
          KQ65
```
Current BBA explains 3-9 HCP.
![image](https://github.com/EdwardPiwowar/BBA/assets/150140020/d986f01e-1d7a-4cf8-96d7-b8255ddb70b6)

**Comment by ThorvaldAagaard** [comment]
Yes, that is wrong, as any 6-card suit, that has no slam-interest and want to play game use Texas, so I expect more like 10-14 hcp - then you can use the 3-9 hcp for Texas.
